src/lib/serviceRoutines/getAttributesForEntityType.cpp

```diff
@@ -49,6 +49,7 @@
 *
 * URI parameters:
 *   - attributesFormat=object
+*   - options=noAttrDetail
```

Comment:
You don't need to trust on me if you can trust on the .test for NGSIv1 type browing operations ;)